docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: '2'
+services:
+    mysql.local:
+        image: keybaseprivate/mysql
+        restart: unless-stopped
+        environment:
+            - MYSQL_ROOT_PASSWORD=secret
+            - MYSQL_DATABASES=gregor
+        ports:
+            - "3306"
+        command: ["--innodb_print_all_deadlocks=1"]
+    kbweb.local:
+        image: keybaseprivate/kbweb
+        ports:
+            - "3000:3000"
+            - "13009"
+            # TODO: remove
```

Comment:
Yeah this will be replaced as soon as keybase/keybase#801 is in, which has already been approved.